notebooks/00_quick_start/rlrmc_movielens.ipynb

```diff
@@ -0,0 +1,267 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Copyright (c) Microsoft Corporation. All rights reserved.*\n",
+    "\n",
+    "*Licensed under the MIT License.*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Riemannian Low-rank Matrix Completion algorithm on Movielens dataset\n",
+    "\n",
+    "Riemannian Low-rank Matrix Completion (RLRMC) is a matrix factorization based (vanilla) matrix completion algorithm that solves the optimization problem using Riemannian conjugate gradients algorithm. \n",
+    "\n",
+    "This notebook provides an example of how to utilize and evaluate RLRMC implementation in **reco_utils**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import sys\n",
+    "import time\n",
+    "import pandas as pd\n",
+    "sys.path.append(\"../../\")\n",
+    "sys.path.append(\"../../reco_utils/recommender/rlrmc/\")\n",
+    "\n",
+    "from reco_utils.dataset.python_splitters import python_random_split\n",
+    "\n",
+    "from reco_utils.dataset import movielens\n",
+    "\n",
+    "from reco_utils.recommender.rlrmc.RLRMCdataset import RLRMCdataset \n",
+    "from reco_utils.recommender.rlrmc.RLRMCalgorithm import RLRMCalgorithm\n",
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Pandas version: 0.24.2\n",
+      "System version: 3.6.7 |Anaconda, Inc.| (default, Oct 23 2018, 19:16:44) \n",
+      "[GCC 7.3.0]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Pandas version: {}\".format(pd.__version__))\n",
+    "print(\"System version: {}\".format(sys.version))\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Select Movielens data size: 100k, 1m, 10m, or 20m\n",
+    "MOVIELENS_DATA_SIZE = '10m'"
```

Comment:
`tags` is missing here. We set tag "parameters" to the cell that contains variables we want to parameterize, so that can pass the values when we call the notebooks via papermill. See [this](https://papermill.readthedocs.io/en/latest/usage-parameterize.html) for how to set.

Reply:
I think you can also move model parameters to this cell, so that we can control **iterations** when we call this notebook from unit-tests.

Reply:
I also added a similar comment, the info on how to set up the tests with papermill is https://github.com/Microsoft/Recommenders/tree/master/tests#how-to-create-tests-on-notebooks-with-papermill

Reply:
done.